From 4809512d9b535a65def2637c9ac03f1561d753fe Mon Sep 17 00:00:00 2001
From: Riddhi Aggarwal <quic_riddagga@quicinc.com>
Date: Thu, 1 Jun 2023 15:37:32 +0530
Subject: [PATCH] arm64: dts: qcom: ipq5332: Adding build support for various
 RDPs

Adding build support for the following RDPs
ipq5332-db-mi01.1.dts
ipq5332-db-mi02.1.dts
ipq5332-db-mi03.1.dts

Change-Id: I9c6bea20d7a18af4ba53a22524ce92b72c083f52
Signed-off-by: Riddhi Aggarwal <quic_riddagga@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 3 +++
 arch/arm64/boot/dts/qcom/Makefile | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 5ffb8b9c8574..a432364546a7 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1077,6 +1077,9 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq5332-rdp468.dtb \
 	ipq5332-rdp472.dtb \
 	ipq5332-rdp474.dtb \
+	ipq5332-db-mi01.1.dtb \
+	ipq5332-db-mi02.1.dtb \
+	ipq5332-db-mi03.1.dtb \
 	qcom-msm8226-samsung-s3ve3g.dtb \
 	qcom-msm8660-surf.dtb \
 	qcom-msm8916-samsung-e5.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 825147e00aa8..ad7ef3aaa88e 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -9,6 +9,9 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp446.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp468.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp472.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp474.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi01.1.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi02.1.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-db-mi03.1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq6018-cp01-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk01.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c1.dtb
-- 
2.34.1

